packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -306,6 +312,24 @@ export const esmPatientRegistrationSchema = {
       _default: '736e8771-e501-4615-bfa7-570c03f4bef5',
     },
   },
+  MPI: {
+    title: {
```

Comment:
If possible, any display strings should **not** be part of the configuration. Just use a reasonable default and use the translation pipeline to translate things. This allows us to support multiple languages while still retaining the ability for implementations to change this to whatever they want.

Reply:
The inspiration of having this config came from the fact that we have mixed terminologies in-regards to the title of the registry. Some folks prefer calling it a "Client Registry"; others simply an MPI. I believe it's imperative to keep this option configurable. cc: @pmanko @ebambo

Reply:
@samuelmale I'm not saying it shouldn't be configurable. I'm just saying it needs to be localisable. If we make this a configuration property, implementors can only ever supply a single string regardless of the number of languages they support. If we use the standard i18n mechanism, we can support both localising it into other languages and having implementation-specific verbiage. This is because translation keys can be provided as part of the application configuration so if I want it to say "Client Registry", I can setup the configuration for this app using [this feature](https://github.com/openmrs/openmrs-esm-core/pull/546).

You're right that this needs to be configurable, but making it part of the config schema like this actually makes it less customisable than it ought to be.

Reply:
Gotcha!! That was detailed @ibacher Thanks!